### DISCUSSION TITLE
Woocommerce add to cart restrictions

### DISCUSSION THREAD

**Discussion by RobertCrx** [bootscore/bootscore#651] [Q&A]
Hello!
I am trying to modify the ```ajax-add-to-cart.php``` in my child to add a new rule before adding to cart. I coudn't find a way to restrict adding products to cart if the product that is added belongs to a certain category and in the cart exists products from another category. Practically I have to find the product category of the current product and compare with the cart items ?

**Comment by crftwrk** [comment]
Hi,

this is something we can't answer because it's very special and goes beyond the default WooCommerce functions.

> Practically I have to find the product category of the current product and compare with the cart items ?

I think yes.
